modules/examples/error-framework/JsExampleErrorRaiser/index.js

```diff
@@ -138,7 +138,7 @@ boot_module(async ({
   mod.provides.example_raiser.raise_error(error);
   check_conditions(mod);
   await sleep(1000);
-  mod.provides.example_raiser.clear_error(error.type, error.sub_type);
+  mod.provides.example_raiser.clear_error(error.type, true);
```

Comment:
FYI: This is not a bug

this will clear at maximum one error that matches type and sub_type:
```
mod.provides.example_raiser.clear_error(error.type, error.sub_type);
```

this will clear multiple errors (in this case 1) that match type
```
mod.provides.example_raiser.clear_error(error.type, true);
```

I would suggest to show both syntax in example module:
```suggestion
  // Clear exactly one error that matches type and sub_type of raised error object
  mod.provides.example_raiser.clear_error(error.type, error.sub_type);
```